package-lock.json

```diff
@@ -64,6 +64,37 @@
         "chalk": "^2.0.0",
         "esutils": "^2.0.2",
         "js-tokens": "^3.0.0"
+      },
```

Comment:
I don't know why this file became this big when I regenerated it... The `package.json` has just a couple of additional packages. Maybe it had not been regenerated for a long time. I am wondering if this file should be in `.gitignore`.

Reply:
The file actually exists exactly for the purpose of checking it into version control, to make sure everyone running `npm install` will get the exact same versions. [More info](https://github.com/npm/npm/blob/v5.0.0/doc/files/package-lock.json.md)